test/test_script.py

```diff
@@ -390,6 +390,45 @@ def test_cmd_with_not_arguments(self):
         except ScriptError:
             self.fail("Function noargs raised ScriptError unexpectedly.")
 
+    def test_cmd_with_wrong_argcount_or(self):
+        # $or() requires at least 2 arguments
+        with self.assertRaises(ScriptError):
```

Comment:
Well, since you're changing the message, it would be great if you would use [assertRaisesRegEx](https://docs.python.org/3/library/unittest.html?highlight=unittest#unittest.TestCase.assertRaisesRegex) to verify the message as well.